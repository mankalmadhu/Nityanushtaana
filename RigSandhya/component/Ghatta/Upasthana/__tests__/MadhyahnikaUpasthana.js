import React from 'react';
import ಮಾಧ್ಯಾಹ್ನಿಕೌಪಸ್ಥಾನ from '../MadhyahnikaUpasthana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../Udutyam',()=>'ಉದುತ್ಯ೦');
jest.mock('../ChitramDevanam',()=>'ಚಿತ್ರ೦ದೇವಾನಾ೦');
jest.mock('../Tachaskshuhu',()=>'ತಚ್ಚಕ್ಷುಃ');
jest.mock('../NamoBramhane',()=>'ನಮೋಬ್ರಹ್ಮಣೇ');
jest.mock('../Pravara',()=>'ಪ್ರವರ');
jest.mock('../AasatyaLokaat',()=>'ಆಸತ್ಯಲೋಕಾತ್');

test("ಮಾಧ್ಯಾಹ್ನಿಕೌಪಸ್ಥಾನ", () => {
   expect(Renderer.create(
    <ಮಾಧ್ಯಾಹ್ನಿಕೌಪಸ್ಥಾನ />
  ).toJSON()).toMatchSnapshot();

});