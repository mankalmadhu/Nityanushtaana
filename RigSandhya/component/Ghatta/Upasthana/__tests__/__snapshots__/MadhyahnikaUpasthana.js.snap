// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಮಾಧ್ಯಾಹ್ನಿಕೌಪಸ್ಥಾನ 1`] = `
<View>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    ಉಪಸ್ಥಾನ
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    (ಪೂರ್ವಾಭಿಮುಖವಾಗಿ ನಿ೦ತು)
  </ವಿಷಯ>
  <ಉದುತ್ಯ೦ />
  <ಚಿತ್ರ೦ದೇವಾನಾ೦ />
  <ತಚ್ಚಕ್ಷುಃ />
  <ನಮೋಬ್ರಹ್ಮಣೇ />
  <ಪ್ರವರ />
  <ಆಸತ್ಯಲೋಕಾತ್ />
</View>
`;
